app/Kernel.php

```diff
@@ -222,7 +222,7 @@ public function getLogDir(): string
 
     public function getCacheDir(): string
     {
-        return dirname(__DIR__) . '/var/cache/' . $this->environment;
+        return dirname(__DIR__) . '/var/cache/' . $this->environment . '/' . $this->request->getHost();
```

Comment:
Won't this decrease the performance? What if I have a site that is available on multiple domains, without a redirect in .htaccess.
Or can you explain why you have a cache per host.

Reply:
this was needed because the cached container parameters are different when the domain is different. Without this it would use the domain that was used to build the cache.
the first time before the cache is built might have a slight difference but it has no effect afterwards. 
Redirects in htaccess won't be needed anymore since that will happen automatically if the url is configured for a language